ext/session/tests/bug79091.phpt

```diff
@@ -50,7 +50,7 @@ class MySessionHandler implements SessionHandlerInterface, SessionIdInterface, S
 
     public function validateId($key)
     {
-        return false;
+        return true;
```

Comment:
This seems concerning. Shouldn't we retain the existing meaning here?

Reply:
That test (written by me) just got it wrong. If a SID is valid (i.e. a respective session already exists), `session_create_id()` has to create a new SID, until it gets an invalid one. The test case is supposed to trigger the create new SID code path.

If we would retain that wrong meaning, session.use_strict_mode would be useless (or actually slight worse that no strict mode), since a none existing session could be created (=> session fixation).